### PR TITLE
Update libp2p-crypto to `^0.22.7`.

### PR DESCRIPTION
**Pull request by paulmillr**
https://github.com/libp2p/js-libp2p-crypto/pull/245

### CODE REVIEW

**Comment by paulmillr** [comment]
@achingbrain ping

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "class-is": "^1.1.0",
-    "libp2p-crypto": "^0.21.0",
+    "libp2p-crypto": "^0.22.8",
```
Comment (wemeetagain) [inline package.json:45]: I don't see a 0.22.x released yet

Comment (paulmillr) [reply package.json:45]: It's there: https://www.npmjs.com/package/@libp2p/crypto

Comment (achingbrain) [reply package.json:45]: `@libp2p/crypto` is ESM only and will break things in this module as it cannot be `require`d.

**Comment by achingbrain** [comment]
`libp2p-crypto@0.21.0` depends on `@noble/ed25519` with a version range so installing it will give you the audited version.

If you want to lock this version down in a distribution of a consuming module, you should ship with a shrinkwrap file.

Are you seeing something different?

**Comment by mpetrunic** [comment]
There was no response in 28 days so closing.

**Pull request closed without merging**